index.bs

```diff
@@ -1318,23 +1340,84 @@ Whenever a [=WebTransport stream=] associated with a {{SendStream}} |stream| get
 
 </div>
 
-# `ReceiveStream` #  {#receive-stream}
+## `WebTransportSendStreamStats` Dictionary ##  {#send-stream-stats}
 
-In this spec, we call a {{ReadableStream}} providing incoming streaming features with an [=incoming
-unidirectional=] or [=bidirectional=] [=WebTransport stream=] a <dfn interface>ReceiveStream</dfn>.
+The <dfn dictionary>WebTransportSendStreamStats</dfn> dictionary includes information
+on stats specific to one {{WebTransportSendStream}}.
+
+<pre class="idl">
+dictionary WebTransportSendStreamStats {
+  DOMHighResTimeStamp timestamp;
+  unsigned long long amountWrittenTo;
+  unsigned long long sentProgress;
```

Comment:
bytesSent ?

Reply:
@martinthomson Sorry I didn't see your comments during the meeting. I wrote "No objections" in the [meeting summary](https://github.com/w3c/webtransport/pull/395#issuecomment-1082476751) 10 minutes later, does that still hold? I think we talked about "bytesSent" briefly, and that it could raise confusion over which layer it was reported from, since "bytes sent" is a term that commonly pops up at different levels of this network stack.

I think we also discussed needing to clarify these were progress counters as opposed to network measurements.
* `bytesSentProgress` was [originally proposed](https://github.com/w3c/webtransport/issues/372#issuecomment-1011536091) alongside a `networkBytesSent` which we dropped
* [Meeting feedback](https://github.com/w3c/webtransport/issues/372#issuecomment-1015538839) was to keep bikeshedding to avoid "confusion over names"
* `sentProgress` was [then proposed](https://github.com/w3c/webtransport/issues/372#issuecomment-1027392673) to avoid any mention of "bytes" and its potential multiple layer meanings.

In the interest of progress, I'd like to merge this PR. Can we open issues on individual member name improvements?